[2024-12-09T15:20:21.693+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T15:20:21.706+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: rtve_schedule_scraper.push_all_in_one scheduled__2024-12-09T05:30:00+00:00 [queued]>
[2024-12-09T15:20:21.713+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: rtve_schedule_scraper.push_all_in_one scheduled__2024-12-09T05:30:00+00:00 [queued]>
[2024-12-09T15:20:21.714+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-09T15:20:21.723+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): push_all_in_one> on 2024-12-09 05:30:00+00:00
[2024-12-09T15:20:21.727+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=526) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T15:20:21.728+0000] {standard_task_runner.py:72} INFO - Started process 527 to run task
[2024-12-09T15:20:21.728+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'rtve_schedule_scraper', 'push_all_in_one', 'scheduled__2024-12-09T05:30:00+00:00', '--job-id', '2220', '--raw', '--subdir', 'DAGS_FOLDER/dag_rtve_selenium.py', '--cfg-path', '/tmp/tmp_n_hfrmx']
[2024-12-09T15:20:21.730+0000] {standard_task_runner.py:105} INFO - Job 2220: Subtask push_all_in_one
[2024-12-09T15:20:21.757+0000] {task_command.py:467} INFO - Running <TaskInstance: rtve_schedule_scraper.push_all_in_one scheduled__2024-12-09T05:30:00+00:00 [running]> on host b605548c7b8b
[2024-12-09T15:20:21.802+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='rtve_schedule_scraper' AIRFLOW_CTX_TASK_ID='push_all_in_one' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T05:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-09T05:30:00+00:00'
[2024-12-09T15:20:21.803+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T15:20:24.227+0000] {logging_mixin.py:190} INFO - Archivo actualizado correctamente.
[2024-12-09T15:20:24.228+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-09T15:20:24.234+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T15:20:24.235+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=rtve_schedule_scraper, task_id=push_all_in_one, run_id=scheduled__2024-12-09T05:30:00+00:00, execution_date=20241209T053000, start_date=20241209T152021, end_date=20241209T152024
[2024-12-09T15:20:24.269+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T15:20:24.278+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-09T15:20:24.279+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
